src/HeaderChainSync/index.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
We should avoid using `index.js` due to many inconveniences. The exception might be only in case if you need dynamic export for your library.